neurobooth_os/netcomm/client.py

```diff
@@ -240,7 +239,7 @@ def start_server(node_name, save_pid_txt=True):
     out = os.popen(cmd_out).read().replace("\\", "")
     df = pd.read_csv(StringIO(out), sep=",", index_col=0, names=["date", "status"])
 
-    task_name = node_name + "1"
+    task_name = node_name + "0"
```

Comment:
I am not familiar with these 1s and 0s. Where does this task name with '0' suffix show up?

Reply:
To go yet another step deeper. It worries me that running another process is not only sketchy engineering, but could cause problems in production if the first process is really live

Reply:
Oh my god - *shivers!*